ooDesign/frankensteinClass/solutionM/src/main/java/oointro/Frankenstein.java

```diff
@@ -0,0 +1,62 @@
+package oointro;
+
+public class Frankenstein {
+    private String cString;     // combination string of song and pet info
+    private double alphaAvg;    // average number of alphanumerics
+    private Song frankenSong;   // aggregate class
+    private Pet frankenPet;     // aggregate class
+
+
+    public void setSong(String song) {
+        // takes string representation of the song and parses it
+        String[] info = song.split(",");
+
+        frankenSong = new Song();
+
+        frankenSong.setTitle(info[0]);
+        frankenSong.setArtist(info[1]);
+        frankenSong.setReleaseYear(Integer.parseInt(info[2]));
+        frankenSong.setRating(Double.parseDouble(info[3]));
+    }
+
+    public String getSong() {
+        return frankenSong.toString();
+    }

```

Comment:
I really like that you've used the toString method instead of just returning the Song.    That's exactly how it should go.